arm/Microsoft.Storage/storageAccounts/.bicep/nested_queue.bicep

```diff
@@ -9,11 +9,11 @@ resource queueObject 'Microsoft.Storage/storageAccounts/queueServices/queues@201
   }
```

Comment:
Suggested
```
param queueObj object
...
resource queue
...
module queue_rbac
```

Reply:
Should now be fixed. Please resolve if you agree.

Reply:
Also updated queue_rbac -> roleAssignment.